ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Run jupyter notebook -&gt; Run jupyter notebook on the command line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>